### ISSUE TITLE
Level: "Forest"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#97]
I had a feeling we're one level short for the outdoors set.
```
:name=Forest
:description=
:num_rabbits=10
:num_to_save=10
:block=2
:climb=4
:dig=4
:bridge=10
:bash=1
:explode=1
:music=tryad-witness
   # ##          
  ######   # #   
   ## #   #####  
           # ##  
   #         /\  
  /#\    #  /##\ 
Q/###\  /#\ /##\ 
 /###\  /#\ /##\ 
   #     #   ## O
#################
```

**Comment by colonelfazackerley** [comment]
I have not had time to try this. It might be better than one of the level in pull request #98

do let me know if something needs changing

**Comment by colonelfazackerley** [comment]
Tricky. Half hour solve time. I think an extra row of sky is needed.

Looks pretty.

**Comment by colonelfazackerley** [comment]
i am adding sky to this for the arcade set,
```
:name=Forest
:description=
:num_rabbits=10
:num_to_save=10
:block=2
:climb=4
:dig=4
:bridge=10
:bash=1
:explode=1
:music=tryad-witness
                           
   # ##          
  ######   # #   
   ## #   #####  
           # ##  
   #         /\  
  /#\    #  /##\ 
Q/###\  /#\ /##\ 
 /###\  /#\ /##\ 
   #     #   ## O
#################
```

**Comment by tttppp** [comment]
Extra row of sky sounds great - I don't think it's going to make it easier, so that's fine in my books. I've had a look for a forest themed arcade game, but none seem to exist. The closest I found was "Jungle Hunt", but the trees are clearly pine trees. I came up with a poor pun for a description:
```
:description=Pining for the exit
```

**Comment by colonelfazackerley** [comment]
Nice. I will put that in.

I can do this level in swing. I have tried for hours to complete it in text mode to get the solution.1. Still have not managed.

**Comment by colonelfazackerley** [comment]
Hallelujah!
```
:solution.1=bash;(2,9);bridge;(12,2);5;(6,9);1;(8,9);9;(6,8);3;(4,6);14;(6,5);9;(7,4);block;(6,5);8;bridge;(6,5);until&WON
```
I would like to see other solutions.

**Comment by colonelfazackerley** [comment]
merged to master f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a
close me

**Comment by andybalaam** [comment]
Took me ages! Lovely-looking level. Not one I stared at for ages, but I kept trying different things that didn't quite work. My final solution uses a bridge on top of the second tree to stop rabbits falling between trees 2 and 3, and involves digging into the cloud twice to allow getting over it.

**Comment by andybalaam** [comment]
The 3rd hint has an extra backslash.

**Comment by andybalaam** [comment]
Hints fixed in 520621268bc72e5f717eb62aaf3c0d5e87138721
